lib/ui/pages/news_list/home_page.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:allthenews/ui/common/util/dimens.dart';
+import 'package:allthenews/ui/common/util/strings.dart';
+import 'package:allthenews/ui/common/widget/primary_text_button.dart';
+import 'package:allthenews/ui/pages/news_list/primary_news/primary_news_list_entity.dart';
+import 'package:allthenews/ui/pages/news_list/primary_news/primary_news_list_view.dart';
+import 'package:allthenews/ui/pages/news_list/secondary_news/secondary_news_list_entity.dart';
+import 'package:allthenews/ui/pages/news_list/news_list_page.dart';
+import 'package:allthenews/ui/pages/news_list/secondary_news/secondary_news_list_view.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const sectionHeaderPadding = 10.0;
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Container(
+              width: double.maxFinite,
+              height: 80,
+            ),
+            _buildNewsSectionHeader(
+              title: Strings.mostViewed,
+              routeBuilder: (context) => NewsListPage(
+                headerTitle: Strings.mostViewed,
+                listEntities: primaryNewsListEntities
+                    .map(
+                      (primaryEntity) => SecondaryNewsListEntity(
+                        title: primaryEntity.title,
+                        date: primaryEntity.date,
+                        imageUrl: primaryEntity.imageUrl,
+                        time: primaryEntity.time,
+                        articleUrl: primaryEntity.articleUrl,
+                      ),
+                    )
+                    .toList(),
+              ),
+            ),
+            SizedBox(height: _Constants.sectionHeaderPadding),
+            PrimaryNewsListView(
+              primaryNewsListEntities: primaryNewsListEntities.take(5).toList(),
+            ),
+            _buildNewsSectionHeader(
+              title: Strings.newest,
+              routeBuilder: (context) => NewsListPage(headerTitle: Strings.newest, listEntities: secondaryNewsListEntities),
```

Comment:
tego wcześniej nie ustalaliśmy, nie wiem jak masz w ustawieniach, ale co ty na to, żeby mieć długość linii na 140 znaków? wiem, że clean code, ale struktury językowe plus specyfika fluttera (zagnieżdżenia) powoduje, że za szybko się te linijki imo zawijają czasem ;)

Reply:
obecnie mam na 160, zmienię na 140